packages/jsii/lib/symbol-id.ts

```diff
@@ -1,9 +1,9 @@
 import { Assembly } from '@jsii/spec';
-import * as fs from 'fs-extra';
+import * as _ from 'lodash';
```

Comment:
Would rather not take a dependency on `lodash`. It's a very large dependency that is capable of WAY MORE than what you're using it for here... That means it's a huge vulnerability surface that we don't even need...

Memoization can be done relatively easily on the user-side... We don't need a dependency for that (`jsii-reflect` has a decorator-based solution for this you can draw inspiration from).